tests/common/plugins/conditional_mark/tests_mark_conditions_drop_packets.yaml

```diff
@@ -62,7 +62,7 @@ drop_packets/test_drop_counters.py::test_dst_ip_link_local:
   skip:
     reason: "Cisco 8000 with 202012 version and broadcom DNX platforms do not drop DIP linklocal packets"
     conditions:
-      - "(asic_type=='cisco-8000' and release in ['202012']) or (asic_subtype in ['broadcom-dnx'])"
+      - "(asic_type=='cisco-8000' and release in ['202012']) or (asic_subtype in ['broadcom-dnx']) or (platform in ['x86_64-8800_lc_48h_o-r0', 'x86_64-8800_lc_48h-r0'])"
```

Comment:
asic_type == 'cisco-8000' should be good enough to skip this. Remove release check as 202012